src/CLI/renderer/templates/show_tacacs_server.j2

```diff
@@ -6,18 +6,20 @@
 {% set vars = {'timeout': ""} %}
 
 {% if json_output -%}
-
 ------------------------------------------------------------------------------------------------
-{{'%-20s'|format("SERVER")}}{{'%-15s'|format("AUTH-TYPE")}}{{'%-10s'|format("KEY")}}{{'%-10s'|format("PORT")}}{{'%-10s'|format("PRIORITY")}}{{'%-10s'|format("TIMEOUT")}}
+{{'%-20s'|format("HOST")}}{{'%-15s'|format("AUTH-TYPE")}}{{'%-10s'|format("KEY")}}{{'%-10s'|format("PORT")}}{{'%-10s'|format("PRIORITY")}}{{'%-10s'|format("TIMEOUT")}}
```

Comment:
Do you want to put a space in between the columns, in case the value in a column exceeds the column width. Otherwise, adjacent columns might show up attached to each other